tmp/build/Dockerfile

```diff
@@ -8,5 +8,17 @@ RUN echo "localhost ansible_connection=local" > /etc/ansible/hosts \
     && echo 'roles_path = /opt/ansible/roles' >> /etc/ansible/ansible.cfg \
     && echo 'library = /usr/share/ansible/openshift' >> /etc/ansible/ansible.cfg
 
+ENV OPERATOR=/usr/local/bin/ansible-operator \
+    USER_UID=1001 \
+    USER_NAME=runner \
```

Comment:
Is `runner` a better username than `ansible-operator`? I'm not opposed necessarily, but just interested in the reasoning since that change doesn't seem related to the rest of the PR.